plugins/woocommerce/src/Blocks/BlockTypes/BlockifiedProductDetails.php

```diff
@@ -35,69 +35,296 @@ protected function get_block_type_style() {
+						return str_replace( '{{title}}', $title, $inner_content );
+					},
+					$block['innerContent']
+				);
+			}
+			if ( 'woocommerce/accordion-panel' === $block['blockName'] ) {
+				$block['innerBlocks']  = parse_blocks( $content );
+				$openning_tag          = reset( $block['innerContent'] );
+				$closing_tag           = end( $block['innerContent'] );
+				$block['innerContent'] = array_merge(
+					array( $openning_tag ),
+					array_fill( 0, count( $block['innerBlocks'] ), null ),
+					array( $closing_tag )
+				);
+			}
+		}
+
+		return $template;
+	}
+
+	/**
+	 * Get the accordion item template.
+	 *
+	 * @param array $parsed_block Parsed block.
+	 *
+	 * @return array Accordion item template.
+	 */
+	private function get_accordion_item_template( $parsed_block ) {
+		$current_accordion_item = $this->get_current_accordion_item( $parsed_block );
+
+		$default_template = parse_blocks(
+			'<!-- wp:woocommerce/accordion-item -->
+			<div class="wp-block-woocommerce-accordion-item"><!-- wp:woocommerce/accordion-header -->
+			<h3 class="wp-block-woocommerce-accordion-header accordion-item__heading"><button class="accordion-item__toggle"><span>{{title}}</span><span class="accordion-item__toggle-icon has-icon-plus" style="width:1.2em;height:1.2em"><svg width="1.2em" height="1.2em" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true"><path d="M11 12.5V17.5H12.5V12.5H17.5V11H12.5V6H11V11H6V12.5H11Z" fill="currentColor"></path></svg></span></button></h3>
+			<!-- /wp:woocommerce/accordion-header -->
+
+			<!-- wp:woocommerce/accordion-panel -->
+			<div class="wp-block-woocommerce-accordion-panel"><div class="accordion-content__wrapper">
+			<!-- wp:paragraph --><p>{{content}}</p><!-- /wp:paragraph -->
+			</div></div>
+			<!-- /wp:woocommerce/accordion-panel --></div>
+			<!-- /wp:woocommerce/accordion-item -->'
+		)[0];
+
+		if ( ! $current_accordion_item ) {
+			return $default_template;
+		}
+
+		foreach ( $current_accordion_item['innerBlocks'] as &$inner_block ) {
+			if ( 'woocommerce/accordion-header' === $inner_block['blockName'] ) {
+				$inner_block['innerContent'] = array_map(
+					function ( $inner_content ) {
+						return preg_replace( '/<span>.*?<\/span>/', '<span>{{title}}</span>', $inner_content );
+					},
+					$inner_block['innerContent']
+				);
+			}
+			/**
+			 * If the accordion panel is empty, the innerContent will be a single string item with
+			 * both opening and closing tags. We need to replace it with the default template for
+			 * create_accordion_item_from_template.
+			 */
+			if ( 'woocommerce/accordion-panel' === $inner_block['blockName'] && empty( $inner_block['innerBlocks'] ) ) {
+				$inner_block['innerContent'] = $default_template['innerBlocks'][1]['innerContent'];
+			}
+		}
+
+		return $current_accordion_item;
+	}
+
+	/**
+	 * Get the current accordion item.
+	 *
+	 * @param array $parsed_block Parsed block.
+	 *
+	 * @return array Current accordion item.
```

Comment:
```suggestion
	 * @return array|false Current accordion item.
```

or you can return an empty array in line 220 instead.

Reply:
I removed this in [9ecc027](https://github.com/woocommerce/woocommerce/pull/57973/commits/9ecc0274884ab568b40fb7c23db5fc72dfa1dda9)